bindings/matlab/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+find_program(
+	MATLAB_EXECUTABLE
+	NAMES matlab
+	PATHS ${Matlab_ROOT_DIR} ${Matlab_ROOT_DIR}/bin
```

Comment:
I was mostly copying straight from the FindMatlab module available in newer cmake versions so its purpose is if a user has multiple matlab versions installed (or has it installed in a non-standard directory) and wants to point specifically at it by setting that variable.

If it's not specified, whatever version is the default or found first is used.